CMSgov/mint-app
# Mail catcher ports

## Changes and Description

- Updated Mail Catcher ports on the mint image.  The mail catcher UI now lives here for MINT http://127.0.0.1:1085/, vs http://127.0.0.1:1080/ for EASi
- also updated the email client config files to reference the updated port for MINT

## How to test this change
1. Run scripts/dev up for both MINT and EASI at the same time. Confirm that you can run the applications concurrently now.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
